jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamDebugSupport.kt

```diff
@@ -73,22 +72,20 @@ class DotNetSamDebugSupport : SamDebugSupport {
         private const val REMOTE_DEBUGGER_DIR = "/tmp/lambci_debug_files"
         private const val REMOTE_NETCORE_CLI_PATH = "/var/lang/bin/dotnet"
         private const val NUMBER_OF_DEBUG_PORTS = 2
+
+        private const val FIND_PID_SCRIPT =
+            """
+                for i in `ls /proc/*/exe` ; do
+                    symlink=`readlink  ${'$'}i 2>/dev/null`;
+                    if [[ " ${'$'}symlink" == *"/dotnet" ]]; then
```

Comment:
is the space here between the double quote and $ important?
```
if [[ " ${'$'}
```

Reply:
Removed